### PR TITLE
update package-security.asciidoc to include link to reset password

### PR DESCRIPTION
**Pull request by andrei-verdes**
Trying to close pull request https://github.com/elastic/elasticsearch/pull/103492 . Seems like it's inactive for a couple of months so I copied the doc changes over here to get it through.

### CODE REVIEW

**Comment by leemthompo** [comment]
run docs-build

**Review by leemthompo — APPROVED** [review]
LGTM thanks!

**Comment by andrei-verdes** [comment]
@leemthompo is the elasticsearch-ci task broken or is someone from the team supposed to merge this?

File: docs/reference/setup/install/package-security.asciidoc [hunk 1]
```diff
@@ -11,6 +11,7 @@ the `elastic` built-in superuser.
 and TLS is enabled and configured with these keys and certificates.
 
 The password and certificate and keys are output to your terminal.
+You can reset the password for the `elastic` user with the <<reset-password,`elasticsearch-reset-password`>> command.
 
 We recommend storing the `elastic` password as an environment variable in your shell. Example:
```
Comment (leemthompo) [inline docs/reference/setup/install/package-security.asciidoc:16]:
```suggestion
We recommend storing the `elastic` password as an environment variable in your shell. For example:
```

**Comment by leemthompo** [comment]
@andrei-verdes CI is acting funny, I'm keeping an eye on this and hopefully we'll get this merged today :)

**Comment by szabosteve** [comment]
@elasticsearchmachine test this please

**Comment by leemthompo** [comment]
Thanks for the contribution @andrei-verdes! And @szabosteve for the CI assist!

**Pull request merged**